libusb/os/sunos_usb.c

```diff
@@ -0,0 +1,1259 @@
+/*
+ *
+ * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please do **not** use "All rights reserved."

I would prefer something like (as suggested by the LGPL license itself):
```
Copyright (C) 2016 leilchen (use your real name and email)
```
"You should also get your employer (if you work as a programmer) or your school, if any, to sign a "copyright disclaimer" for the library, if necessary."
See https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html#SEC4

Thanks